tests/dialects/test_oracle.py

```diff
@@ -1,11 +1,16 @@
 from sqlglot import exp
 from sqlglot.errors import UnsupportedError
 from tests.dialects.test_dialect import Validator
+from sqlglot.dialects.oracle import eliminate_join_marks
 
 
 class TestOracle(Validator):
     dialect = "oracle"
 
+    def validate(self, transform, sql, target):
```

Comment:
let's delete this helper method and just do a for loop with sub tests

for in, out in [(..., ...), ...]

Reply:
removed helper and converted to loop